test/functional/test_framework/test_node.py

```diff
@@ -232,8 +232,18 @@ def wait_for_rpc_connection(self):
         poll_per_s = 4
         for _ in range(poll_per_s * self.rpc_timeout):
             if self.process.poll() is not None:
+                str_error = ""
+                if self.process.returncode != 0:
```

Comment:
Merely to not print the "********\n" string when there is no error. But just pushed a slightly different version only checking the stderr instead of the return code.